generators/server/templates/src/main/java/package/config/DatabaseConfiguration_couchbase.java.ejs

```diff
@@ -164,25 +165,34 @@ public class DatabaseConfiguration extends AbstractCouchbaseConfiguration {
         return mappingCouchbaseConverter;
     }
 
-    /**
-     * Workaround for Couchbase overriding SB default jackson mapper: https://github.com/spring-projects/spring-data-couchbase/issues/1209
-     */
-    @Bean
-    @Primary
-    ObjectMapper jacksonObjectMapper(Jackson2ObjectMapperBuilder builder) {
-        return builder.createXmlMapper(false).build();
-    }
-
     @Bean
     public Couchmove couchmove(Cluster cluster) {
         log.debug("Configuring Couchmove");
         Bucket bucket = cluster.bucket(getBucketName());
-        Couchmove couchmove = new Couchmove(bucket, cluster, "config/couchmove/changelog");
+        Collection collection = bucket.scope(getScopeName()).collection(CHANGELOG_COLLECTION);
+        Couchmove couchmove = new Couchmove(collection, cluster, "config/couchmove/changelog");
         couchmove.migrate();
-        couchmove.buildN1qlDeferredIndexes();
+        couchmove.buildN1qlDeferredIndexes(getScopeName());
         return couchmove;
     }
 
+    @Bean
+    @Profile(JHipsterConstants.SPRING_PROFILE_TEST)
```

Comment:
```suggestion
    @Profile({JHipsterConstants.SPRING_PROFILE_TEST, JHipsterConstants.SPRING_PROFILE_E2E})
```

`SPRING_PROFILE_E2E` doesn't exist yet.

Reply:
If you are happy with the proposed solution, I’ll add `SPRING_PROFILE_E2E ` to jhipster-framework